src/network/netdev/netdev.c

```diff
@@ -760,8 +759,8 @@ int netdev_load_one(Manager *manager, const char *filename) {
                 assert(n);
                 if (!streq(netdev->filename, n->filename))
                         log_netdev_warning_errno(netdev, r,
-                                                 "The setting Name=%s in %s conflicts with the one in %s, ignoring",
-                                                 netdev->ifname, netdev->filename, n->filename);
+                                                 "The file %s conflicts with %s, ignoring",
+                                                 netdev->filename, n->filename);
```

Comment:
IIUC when this error happens, the original message wasn't too good either. But it's important that they both refer to the same iface, i.e. to the same Name. Maybe say: "Device was already configured by file %s, ignoring %s." ?